typhos/utils.py

```diff
@@ -228,38 +234,104 @@ def clean_name(device, strip_parent=True):
     return clean_attr(name)
 
 
-def use_stylesheet(dark=False, widget=None):
+def use_stylesheet(
+    dark: bool = False,
+    widget: QtWidgets.QWidget | None = None,
+) -> None:
     """
     Use the Typhos stylesheet
 
+    This is no longer used directly in typhos in favor of compose_stylesheets.
+
     Parameters
     ----------
     dark: bool, optional
         Whether or not to use the QDarkStyleSheet theme. By default the light
         theme is chosen.
     """
-    # Dark Style
+    style = typhos_style_string(dark=dark)
+    if widget is None:
+        widget = QtWidgets.QApplication.instance()
+    # We can set Fusion style if it is an application
+    if isinstance(widget, QtWidgets.QApplication):
+        widget.setStyle(QtWidgets.QStyleFactory.create('Fusion'))
+
+    # Set Stylesheet
+    widget.setStyleSheet(style)
+
+
+def typhos_style_string(dark: bool = False) -> str:
+    """Load the typhos style string."""
     if dark:
         import qdarkstyle
-        style = qdarkstyle.load_stylesheet_pyqt5()
-    # Light Style
+        return qdarkstyle.load_stylesheet_pyqt5()
     else:
-        # Load the path to the file
         style_path = os.path.join(ui_dir, 'style.qss')
         if not os.path.exists(style_path):
             raise OSError("Unable to find Typhos stylesheet in {}"
                           "".format(style_path))
-        # Load the stylesheet from the file
         with open(style_path) as handle:
-            style = handle.read()
-    if widget is None:
-        widget = QtWidgets.QApplication.instance()
-    # We can set Fusion style if it is an application
+            return handle.read()
+
+
+def compose_stylesheets(
```

Comment:
Code reusability-wise, I'd have expected that `compose_stylesheets` would merely have combined the source code so that you could do with it as you please.

A more apt name might be "compose stylesheets and apply to maybe a widget or the entire application" which makes it sound like it's doing a bit too much.

Reply:
I agree.

It turns out that the only thing you need to do to combine two stylesheets is to append them together, but having a function to do that and to abstract away the priority order could be useful.

Reply:
I'll rename and refactor out the order/priority bits to a smaller function with this original name.

Reply:
For more context: there are related utilities in https://github.com/slaclab/pydm/blob/master/pydm/utilities/stylesheet.py but they are a bit too specific to be used as-is here